lib/inspec/resources/virtualization.rb

```diff
@@ -236,6 +254,39 @@ def detect_lxd
       true
     end
 
+    # Detect VMware
```

Comment:
These should be added to the list of possible virtualisation systems in the documentation